features/openhab-core/src/main/feature/feature.xml

```diff
@@ -75,15 +80,6 @@
         <bundle dependency="true">mvn:org.eclipse.jetty.websocket/websocket-client/${jetty.version}</bundle>
     </feature>
 
-    <feature name="openhab-transport-coap" description="CoAP Transport" version="${project.version}">
-        <feature>esh-io-transport-coap</feature>
-    </feature>
-
-    <feature name="openhab-transport-http" description="HTTP Transport" version="${project.version}">
-        <feature>openhab-httpclient</feature>
-        <feature>esh-io-transport-http</feature>
```

Comment:
Is this now still mentioned?

Reply:
No, but it also rather causes issues as it refers to Jetty 9.3.15 - so it should be safer to leave it out.